monix-execution/jvm/src/main/scala/monix/execution/schedulers/TracingScheduler.scala

```diff
@@ -0,0 +1,89 @@
+package monix.execution.schedulers
+
+import java.util.concurrent.{ScheduledExecutorService, TimeUnit}
+import monix.execution.misc.Local
+import monix.execution.{Cancelable, Scheduler, UncaughtExceptionReporter, ExecutionModel => ExecModel}
+import scala.concurrent.ExecutionContext
+
+final class TracingScheduler private (
+                                       scheduler: ScheduledExecutorService,
+                                       ec: ExecutionContext,
+                                       r: UncaughtExceptionReporter,
+                                       val executionModel: ExecModel)
```

Comment:
Please fix formatting. You can configure IntelliJ IDEA to not align parameters like this.